test_rclcpp/test/node_check_names.cpp

```diff
@@ -40,16 +43,23 @@ int main(int argc, char ** argv)
   int counter = 0;
   const std::chrono::steady_clock::time_point max_runtime =
     std::chrono::steady_clock::now() + std::chrono::seconds(10);
-  std::string node_to_look_for("node_with_name");
+
   while (rclcpp::ok()) {
-    printf("\n");
-    auto names = node->get_node_graph_interface()->get_node_names();
+    auto names_namespaces = node->get_node_graph_interface()->get_node_names_and_namespaces();
+    std::vector<std::string> names;
+    std::transform(names_namespaces.begin(),
+      names_namespaces.end(),
+      std::back_inserter(names),
+      [](std::pair<std::string, std::string> nns) {return nns.first;}
+    );
+    printf("BEGIN DISCOVERED NODES\n");
```

Comment:
Looks like debug messages, can you remove those please?

Reply:
Whoops, on it.